lib/service/Service.ts

```diff
@@ -269,7 +270,7 @@ class Service {
       nodePubKey,
       uris,
       numPairs,
-      version: this.version,
+      version: `${this.version}${commitHash}`,
```

Comment:
I would go for `${this.version}${commitHash}`, since imo the version name should only be on the official release.

`version` in `xucli getinfo -j` should look like: `master-9a03442c-dirty`

Reply:
Are you sure about that? With the current approach if the commit is also the tag it will just show the version from `package.json`. If not it will give you the `package.json` version and the commit hash. This is enough information to know whether you are running the release I think.

Also it is really nice to know on what release the branch you are running is based on.

Reply:
> Also it is really nice to know on what release the branch you are running is based on.

Ok, I can see that. Fine then.